### DISCUSSION TITLE
use environment variable for connection string on config file

### DISCUSSION THREAD

**Discussion by febridev** [burningalchemist/sql_exporter#445] [Q&A] [answered]
hi
I have mention from security team about connection string  at `sql_exporter.yml ` is explicit, how to make username and password on connection string is set from variable environment 
because we can't put explicitly on that's config.

**Comment by burningalchemist** [comment, marked as answer]
Hi @febridev , please have a look at this discussion topic: https://github.com/burningalchemist/sql_exporter/discussions/444#discussioncomment-8405151

I'll include the section in the documentation 🙂👍

**Reply by febridev** [reply]
thank you very much
